tutorials/packaging/continuous-snap-delivery-from-travis-ci/continuous-snap-delivery-from-travis-ci.md

```diff
@@ -113,38 +115,38 @@ We now have a GitHub project containing packaging details for a snap and it’s
+To use Travis CI, first we will have to sign up there. Let’s go to [https://travis-ci.com/auth] and click "Sign in with GitHub". Then, click "Authorize application" to allow Travis to access your GitHub account.
 
-Now, enable the GitHub repository of your snap by flipping the switch next to it at [https://travis-ci.org/profile/]
+Now, enable the GitHub repository of your snap by clicking the "Activate repositories on GitHub" button at [[https://travis-ci.com/profile/], and select the repositories you want to use with Travis CI on [https://github.com/apps/travis-ci]
 
-![](images/3.png)
+![Add the GitHub repository to Travis CI](images/3-add-GitHub-repos-to-travis.png)
 
-We need a `.travis.yml` file for your project. Travis CI will run the steps specified in this file on every pull request and every time a change is pushed to the master branch. These steps are going to be a few bash commands and scripts.
+We need a `.travis.yml` file for your project. Travis CI will run the steps specified in this file on every pull request and every time we make a change to any of the branches. These steps will be a few bash commands and scripts to run your tests and build your code.
 
 First, create a file named `.travis.yml` in your project folder, with the following content:
 
 ``` bash
 language: bash
 ```
 
-Later you can use Travis CI to run your tests. For that you will have to adjust this file depending on the language of your project, and add the install and script steps. But for now, we don’t need any of that as we’ll just make continuous delivery to the store.
+Later you can use Travis CI to run your tests. For that you will have to adjust this file depending on the language of your project, and add the install and script steps to test and build your snap before publishing. However, we don’t need any of that yet - as we’ll just focus on continuous delivery to the store.
 
-In the next step we’ll make sure that your snap can be built successfully in Travis CI, so we can push the generated snap to the store.
+In the next step, we’ll make sure that your snap can be built successfully in Travis CI, so we can push the generated snap to the store.
 
 ## Building the snap in a clean environment
 Duration: 5:00
 
-Travis CI uses a build environment based on Ubuntu 14.04, which is missing some requirements for running snapcraft. However, it does have support for Docker, so we will use a Docker container to build the snap.
+Travis CI uses a [build environment based on Ubuntu 16.04](https://docs.travis-ci.com/user/reference/overview/) and it has Docker support, so we will use a Docker container to build the snap.
 
 Before we build our snap in Travis CI’s executors, we need to make sure the snap builds correctly in a Docker container.
 
 Positive
 : **The need for a clean environment**
 When you build a snap on your machine, it could be using some dependencies that are already installed but you forgot to declare as `build-packages` in the snapcraft.yaml.
```

Comment:
```suggestion
When you build a snap on your machine, it could be using some dependencies that are already installed but you forgot to declare as `build-packages` in the `snapcraft.yaml`.
```